streams/src/test/resources/log4j.properties

```diff
@@ -29,6 +29,8 @@ log4j.logger.org.apache.kafka.clients.consumer=INFO
 log4j.logger.org.apache.kafka.clients.producer=INFO
 log4j.logger.org.apache.kafka.streams=INFO
 log4j.logger.org.apache.kafka.coordinator.group=INFO
+log4j.logger.org.apache.kafka.clients.consumer.internals.StreamsGroupInitializeRequestManager=INFO
+log4j.logger.org.apache.kafka.clients.consumer.internals.StreamsGroupHeartbeatRequestManager=INFO
```

Comment:
Are these remainings of a debug session?

Reply:
Yes, but I decided to keep them. We do want to show INFO logs of all streams-specific classes in the tests, so it's good to show these two.